### ISSUE TITLE
7f8a8) :  DEBUG (services:618) - No matching services found for 

### ISSUE THREAD

**Issue by txt3rob** [Cigaras/IPTV.bundle#23]
Any ideas why i get this error in the log?

from the playlist
```
#EXTINF:-1,some video
http://serverurl/6013.mkv
```

whats in the error log
```
2015-11-26 15:10:38,121 (7f8a8) :  DEBUG (services:618) - No matching services found for 'http://serverurl/6013.mkv'
```

**Comment by Cigaras** [comment]
Client is unable to transcode/play the stream, server should do the transcoding but it does it only for local videos, more info can be found in readme file section [Compatible devices and limitations](https://github.com/Cigaras/IPTV.bundle#compatible-devices-and-limitations).

**Comment by brickibezz** [comment]
hello sir, i am a great fan of your iptv.bundle and would like to make a donation to you as i use it ever day and you have saved me alot of money, also i would like to chat to you about the epg do you think this would be possible? if not i have an idea for developing a new channel for it be good to hear from you my email is brickibezz@hotmail.co.uk

**Comment by Cigaras** [comment]
EPG is available (kind of) in version 1.1.0, more info in the [readme file](https://github.com/Cigaras/IPTV.bundle#program-guide) and [forum](http://forums.plex.tv/discussion/comment/1152230/#Comment_1152230).
